Makefile

```diff
@@ -12,14 +12,17 @@ all: check container
 lint: essentials.yaml shunit2
 	./tools/yamllint -s ./essentials.yaml
 	./tools/shellcheck -x tests/tests.sh
-	./tools/shellcheck -x scripts/shim-startup-wrapper.sh
+	./tools/shellcheck -x scripts/*.sh
 
 check: lint
 	$(MAKE) -C client $@
 	$(MAKE) -C services $@
 	$(MAKE) container-check
 
-container-prereqs: build/jenkins-support build/jenkins.sh scripts/shim-startup-wrapper.sh build/configuration-as-code/target/configuration-as-code.hpi build/essentials/target/essentials.hpi
+container-prereqs: build/jenkins-support build/jenkins.sh scripts/shim-startup-wrapper.sh build-plugins
+
+build-plugins:
```

Comment:
This should be added as a PHONY target to the bottom of the file

Reply:
done